Source/Core/Core/HW/WiimoteEmu/Extension/BalanceBoard.cpp

```diff
@@ -20,29 +20,45 @@
 
 #include "InputCommon/ControllerEmu/Control/Input.h"
 #include "InputCommon/ControllerEmu/ControlGroup/AnalogStick.h"
-#include "InputCommon/ControllerEmu/ControlGroup/Buttons.h"
 #include "InputCommon/ControllerEmu/ControlGroup/ControlGroup.h"
-#include "InputCommon/ControllerEmu/ControlGroup/Force.h"
-#include "InputCommon/ControllerEmu/ControlGroup/Tilt.h"
+#include "InputCommon/ControllerEmu/ControlGroup/Triggers.h"
 
 namespace WiimoteEmu
 {
 constexpr std::array<u8, 6> balance_board_id{{0x00, 0x00, 0xa4, 0x20, 0x04, 0x02}};
 
 BalanceBoard::BalanceBoard() : Extension1stParty("BalanceBoard", _trans("Balance Board"))
 {
+  // balance
+  groups.emplace_back(
+      m_balance = new ControllerEmu::AnalogStick(
+          _trans("Balance"), std::make_unique<ControllerEmu::SquareStickGate>(1.0)));
+  groups.emplace_back(m_weight = new ControllerEmu::Triggers(_trans("Weight")));
+  m_weight->controls.emplace_back(
+      new ControllerEmu::Input(ControllerEmu::Translatability::Translate, _trans("Weight")));
 }
 
 void BalanceBoard::BuildDesiredExtensionState(DesiredExtensionState* target_state)
 {
+  const ControllerEmu::AnalogStick::StateData balance_state = m_balance->GetState();
+  const ControllerEmu::Triggers::StateData weight_state = m_weight->GetState();
+  const auto weight = weight_state.data[0];
+
+  const double total_weight = 63.5 * weight;  // kilograms
```

Comment:
Nope, I can't. I don't think it has any special meaning, other than being chosen as a default weight of some sort (because the way I implemented the controller UI, it gives a value from 0x to 2x instead of a weight in kilograms). It looks like I created a `DEFAULT_WEIGHT` constant in WiiTASInputWindow.cpp but just left it as a magic number here. I have no idea why I picked 63.5 though.